.ci/release-pipeline.yml

```diff
@@ -14,9 +14,14 @@ jobs:
   steps:
   - script: |
       echo "##vso[task.setvariable variable=version]$(git tag --points-at HEAD)"
-  - template: templates/publish.yml
-    parameters:
-      version: $(version)
+  - bash: ./gradlew --no-daemon -Pversion=$GRADLE_VERSION publish
+    env:
+      GRADLE_VERSION: $(version)
       ${{ if not(endsWith(variables['version'], '.RELEASE')) }}:
-        repoId: libs-milestone-local
-      # TODO support release publishing
+        GRADLE_PUBLISH_REPO_URL: https://repo.spring.io/libs-milestone-local/
+        GRADLE_PUBLISH_MAVEN_USER: $(artifactory.username)
+        GRADLE_PUBLISH_MAVEN_PASSWORD: $(artifactory.password)
+      ${{ if endsWith(variables['version'], '.RELEASE') }}:
+        GRADLE_PUBLISH_REPO_URL: https://api.bintray.com/maven/spring/jars/BlockHound/
```

Comment:
This isn't right, I think. for instance pool is released into `bintray.com/spring/jars/io.projectreactor.addons/`, so here it should be `io.projectreactor.tools` instead of `BlockHound`? Not sure how the publish plugin is configured, but the folder structure root should be at least modified